bedrock/settings/static_media.py

```diff
@@ -859,6 +859,13 @@
         ),
         'output_filename': 'css/facebookapps_downloadtab-bundle.css',
     },
+    'thunderbird-organizations': {
+        'source_filenames': (
+            'css/base/menu.less',
+            'css/firefox/organizations.less',
+        ),
+        'output_filename': 'css/thunderbird-organizations-bundle.css',
```

Comment:
Nit: Naming convention would dictate this to be `thunderbird_organizations-bundle.css`